app/src/free/java/com/blogspot/e_kanivets/moneytracker/util/CrashlyticsProxy.java

```diff
@@ -0,0 +1,47 @@
+package com.blogspot.e_kanivets.moneytracker.util;
+
+import android.content.Context;
+import android.support.annotation.Nullable;
+
+
+/**
+ * In free software builds, this class does nothing as Crashlytics can't be used.
+ * Created on 28/8/19.
+ *
+ * @author Fynn Godau
+ */
+
+public class CrashlyticsProxy {
+    private static CrashlyticsProxy instance;
```

Comment:
This singleton implementation is not thread safe. Could be improved with double-checked locking.